### PR TITLE
Add command `awspec generate lambda`

### PR DESCRIPTION
**Pull request by k1LoW**

### CODE REVIEW

File: lib/awspec/stub/lambda.rb [hunk 1]
```diff
@@ -5,9 +5,12 @@
         {
           function_name: 'my-lambda-function-name',
           function_arn: 'arn:aws:lambda:us-east-1:123456789012:function:my-lambda-function-name',
-          runtime: 'nodejs',
+          description: 'My Lambda Function',
+          runtime: 'pyhton2.7',
```
Comment (neil-greenwood) [inline lib/awspec/stub/lambda.rb]: Typo.

File: spec/generator/spec/lambda_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    Awspec::Stub.load 'lambda'
+  end
+  let(:lambda) { Awspec::Generator::Spec::Lambda.new }
+  it 'generates spec' do
+    spec = <<-'EOF'
+
+describe lambda('my-lambda-function-name') do
+  it { should exist }
+  its(:description) { should eq 'My Lambda Function' }
+  its(:runtime) { should eq 'pyhton2.7' }
```
Comment (neil-greenwood) [inline spec/generator/spec/lambda_spec.rb]: Typo

File: spec/type/lambda_spec.rb [hunk 3]
```diff
@@ -3,8 +3,10 @@
 
 describe lambda('my-lambda-function-name') do
   it { should exist }
-  its(:function_name) { should eq 'my-lambda-function-name' }
-  its(:runtime) { should eq 'nodejs' }
+  its(:description) { should eq 'My Lambda Function' }
+  its(:runtime) { should eq 'pyhton2.7' }
```
Comment (neil-greenwood) [inline spec/type/lambda_spec.rb]: And again. If it's intentional, add a comment or use something that is less similar to a real word.

File: lib/awspec/stub/lambda.rb (hunk 1 above)
Comment (k1LoW) [inline lib/awspec/stub/lambda.rb]: Opps!! Thank you!

**Comment by k1LoW** [comment]
Released as v0.34.0.

**Pull request merged**